### DISCUSSION TITLE
Can not access Admin UI

### DISCUSSION THREAD

**Discussion by lilran14** [pocketbase/pocketbase#2671] [Q&A] [answered]
I installed PocketBase with docker on port 8080, when i access the url its redirect to ":8080/_/?installer#" and stuck on that. What am I missing? Thank you.

server :
Linux Ubuntu 22.04 X86 64
![image](https://github.com/pocketbase/pocketbase/assets/85271559/8f0e005d-fc53-458f-b001-ad371144a335)

**Comment by ganigeorgiev** [comment]
Could you provide a little more details about your setup and/or share your Dockerfile?

Note that by default PocketBase binds to the `localhost/127.0.0.1` interface meaning that it will be visible only within the Docker container. Dependending on your setup you can start PocketBase with `--http=0.0.0.0:8080` to bind to all IPv4 interfaces, something like:

```dockerfile
CMD ["/pb/pocketbase", "serve", "--http=0.0.0.0:8080"]
```

**Reply by lilran14** [reply]
I used the same Dockerfile as on the documentation

```
FROM alpine:latest

ARG PB_VERSION=0.16.4

RUN apk add --no-cache \
    unzip \
    ca-certificates

# download and unzip PocketBase
ADD https://github.com/pocketbase/pocketbase/releases/download/v${PB_VERSION}/pocketbase_${PB_VERSION}_linux_amd64.zip /tmp/pb.zip
RUN unzip /tmp/pb.zip -d /pb/

EXPOSE 8080

# start PocketBase
CMD ["/pb/pocketbase", "serve", "--http=0.0.0.0:8080"]
```

**Reply by ganigeorgiev** [reply]
I'm not sure how to reproduce it. Testing locally with the above Dockerfile is working fine for me.
Do you have a reverse proxy in your setup (eg. nginx)? Could you elaborate a little more about your setup configuration? Any info or reproducible steps are helpful.

**Reply by lilran14** [reply]
I installed on fresh OS, maybe there is something to install before?

**Reply by lilran14** [reply, marked as answer]
its working now after I add a domain to my service, I close this discussion, thank you.

**Reply by appelskrutt34** [reply]
What do you mean by adding a domain to your service? I have the same problem and need to know how you solved it

**Reply by lilran14** [reply]
did you publish it to a hosting? If so, you need to add a domain with SSL and navigate to your PocketBase IP. Then, you can access it.

My setup:

Install PocketBase on Docker.
Navigate your domain to your IP.
Add your domain with Nginx Proxy Manager.

**Reply by appelskrutt34** [reply]
Im just running it locally in docker with "docker run -it --rm -p 8080:8080 pocketbase"

**Reply by lilran14** [reply]
Pocketbase runs on port 8090. If you want to access it through port 8080, change your command to:
`docker run -it --rm -p 8080:8090 pocketbase`

**Reply by ganigeorgiev** [reply]
@lilran14 That's not true if you use the above Dockerfile.

@appelskrutt34 Please share your Dockerfile and provide more details about your environment and how do you access the application so that I can try to reproduce it.
Also note that if you just want to run PocketBase you don't really need Docker or anything else, just download the zip for your platform from the [releases](https://github.com/pocketbase/pocketbase/releases) and run the executable inside it.

**Reply by lilran14** [reply]
Yes @ganigeorgiev sir, I forgot to mention that if you use the Docker CLI and run it from either of these images:
1. https://github.com/muchobien/pocketbase-docker
2. https://hub.docker.com/r/augustodelg/pocketbase

This command will work:
`docker run -it --rm -p 8080:8090 pocketbase`
